packages/legacy-client/src/oauth-client/PublicClient/PublicClientAuth.ts

```diff
@@ -33,7 +33,7 @@ import {
 
 export class PublicClientAuth implements Auth {
   private palantirRefreshToken = "palantir_refresh_token" as const;
-  private palantirPcke = "palantir_pcke" as const;
+  private palantirPkce = "palantir_pkce" as const;
```

Comment:
The rename here and the move from local to session storage does leave behind old values but given that this state should only live for the lifetime of one auth flow there shouldn't be a risk of breaking a user's login